globus_sdk/auth/token_response.py

```diff
@@ -56,11 +60,19 @@ def by_resource_server(self):
         return self._by_resource_server
 
     @property
+    def expires_at(self):
+        """
+        A POSIX timestamp representation of the expiration
+        time for the top-level token in the response.
+        """
+        return self['expires_at']
```

Comment:
Oops--you're right.

I agree that the top-level properties are sort of strange. But as you mention below, they do come in useful for quickly grabbing something like `expires_in`.

Although as discussed yesterday, if and when we end up in a world where not all tokens have the same `expires_in` value even that small benefit goes away.